examples/yaml/control_rrfs_cmaq_airnow_reg.yaml

```diff
@@ -0,0 +1,246 @@
+    files: /scratch2/BMC/rcm1/rhs/fv3/regional/processed/v150_a/data/rrfs_v150_a/201908*dynf*.nc
+    files_pm25: /scratch2/BMC/rcm1/rhs/fv3/regional/processed/v150_a/data/rrfs_v150_a/201908*pm25tot*.nc
+    mod_type: 'rrfs'
+    mod_kwargs:
+      mech: 'cb6r3_ae6_aq'
+      surf_only: True
+    radius_of_influence: 19500 #meters: horizontal resolution * 1.5
+    mapping: #model species name : obs species name
+      airnow:
+        #co: 'CO'
+        #no2: 'NO2'
+        #'no': 'NO' #Be careful any python commands like no need to be sent in as a string.
+        PM25_TOT: 'PM2.5'
+        o3: 'OZONE'
+    projection: None
+    plot_kwargs: #Opt
+      color: 'goldenrod'
+      marker: '^'
+      linestyle: '-'      
+obs:
+  airnow: # obs label
+    use_airnow: True
+    filename: /scratch2/BMC/rcm1/rhs/obs/Surface/MELODIES-MONET/AIRNOW_20190801_20190831.nc
+    obs_type: pt_sfc
+    variables: #Opt 
+      PM2.5:
+        unit_scale: 1
+        unit_scale_method: '*' # Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        #obs_min: 0 # set all values less than this value to NaN
+        #obs_max: 100 # set all values greater than this value to NaN
+        nan_value: -1.0 # Set this value to NaN
+        #The obs_min, obs_max, and nan_values are set to NaN first and then the unit conversion is applied.
+        ylabel_plot: 'PM2.5 (ug/m3)' #Optional to set ylabel so can include units and/or instr etc.
+        ty_scale: 3.0 #Opt
+        vmin_plot: 0.0 #Opt Min for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vmax_plot: 22.0 #Opt Max for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vdiff_plot: 15.0 #Opt +/- range to use in bias plots. To apply to a plot, change restrict_yaxis = True.
+        nlevels_plot: 23 #Opt number of levels used in colorbar for contourf plot.
+        regulatory: True #Opt compute regulatory functions
+        ylabel_reg_plot: 'PM2.5_24hr (ug/m3)'
+        vmin_reg_plot: 0.0 #Opt
+        vmax_reg_plot: 22.0 #Opt
+        vdiff_reg_plot: 5.0 #Opt
+      OZONE:
+        unit_scale: 1 #Opt Scaling factor 
+        unit_scale_method: '*' #Opt Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        nan_value: -1.0 # Opt Set this value to NaN
+        ylabel_plot: 'Ozone (ppbv)'
+        vmin_plot: 15.0 #Opt Min for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vmax_plot: 55.0 #Opt Max for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vdiff_plot: 20.0 #Opt +/- range to use in bias plots. To apply to a plot, change restrict_yaxis = True.
+        nlevels_plot: 21 #Opt number of levels used in colorbar for contourf plot.
+        regulatory: True #Opt compute regulatory functions
+        ylabel_reg_plot: 'MDA8 O3 (ppbv)'
+        percentile_opt: 0.95 #Opt, 0.05, 0.5, 0.95, etc. If not specified, simply calcualte mean values. Used for spatial plots only.
```

Comment:
@jianheRCM can we change the location for this percentile_opt. It fits better in the plots section under data_proc. For example, all the other plotting/data analysis type metrics that are not variable specific are stored here instead. This would make it nice because if someone wanted to plot mean, 95th, and 99th percentile spatial bias plots they could just do this with one submit script and have three plotting groups with the different options. Does this make sense?

Reply:
Good point. Will put under data_proc.